docs/data-structures.rst

```diff
@@ -128,54 +114,38 @@ Gavel Validator Output (object)
 
```

Comment:
I'd rename this to "Gavel Validation Result Field". To keep permalinks working, you can do this:

```rst
.. _gavel-validator-output:
.. _gavel-validation-result-field:

Gavel Validation Result Field (object)
--------------------------------------
```